target/linux/realtek/dts-5.10/rtl9302_zyxel_xgs1010-12.dts

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: (GPL-2.0-or-later or MIT)
+/dts-v1/;
+
+#include "rtl9302_zyxel_xgs1210-12.dts"
+
+/ {
+	compatible = "zyxel,xgs1010-12", "realtek,rtl838x-soc";
+	model = "Zyxel XGS1010-12 Switch";
+};
+
+&partitions {
+	compatible = "fixed-partitions";
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	partition@300000 {
+		label = "firmware";
+		reg = <0x300000 0x680000>;
+		compatible = "openwrt,uimage", "denx,uimage";
```

Comment:
```suggestion
```

the only reason we overwrite this partition, is because of the ih-magic. If anybody has any idea how to put the content in some variable, or alias a property; or derivce it some other way from, that would be much nicer :)